my-app/src/components/beachspeech/BSGameLobby.js

```diff
@@ -0,0 +1,52 @@
+import React from "react"
+import "../../stylesheets/beachspeech.css"
+import Island from "./Island"
+import CenterColumn from "./CenterColumn"
+import RightColumn from "./RightColumn"
+import Developers from "./Developers"
+import playerData from "../../data/beachspeech/playerData.json"
+
+/* Beach Speech Game Lobby */
+class BSGameLobby extends React.Component {
+    constructor() {
+        super()
+        this.state = {
+            gameCode: "ABCD",
```

Comment:
Just a question about gameCode, how do we plan on getting this variable? Tts out of ur ticket's scope but do we want to pass it in when this is being called or do we want to do something like getting the value from the backend then update the state.

Reply:
I was thinking about getting the gameCode from the backend when it "creates a room"